security.md

```diff
@@ -0,0 +1,7 @@
+Contact: mailto:security@ubuntu.com
+Expires: 2025-08-01T00:00:00.000Z
```

Comment:
Not sure how is this date defined, but it's not too far into the future. Can we move it further ahead?

Reply:
I've extended it to the end of the year. It will be reviewed by the security team too so if it needs changing beyond that it shouldn't be an issue.